### PR TITLE
Fix for ticket # 3644

### PR DESCRIPTION
**Pull request by washu**
sorry for 2 files, pushed it up befor ei took out the comment. This will fix the above ticket issue.

### CODE REVIEW

**Comment by washu** [comment]
any idea if this wil be merged in?

File: spec/app/models/package.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Package
```
Comment (arthurnn) [inline spec/app/models/package.rb]: do we need these new classes? 
Can we use an existent model?

File: spec/mongoid/relations/referenced/many_spec.rb [hunk 2]
```diff
@@ -3443,7 +3443,27 @@
       end
     end
   end
-
+  
+  context "when a document has embedded and references it saves correctly" do
+    let(:container) do
```
Comment (arthurnn) [inline spec/mongoid/relations/referenced/many_spec.rb]: can you follow the spacing patterns from this test file?
thanks

**Comment by estolfo** [comment]
no longer an issue: https://jira.mongodb.org/browse/MONGOID-3644

**Pull request closed without merging**